test/utils/utils.test.js

```diff
@@ -229,8 +229,8 @@ describe('Utils', () => {
         config.locales = {
           '': { ietf: 'en-US', tk: 'hah7vzn.css' },
           africa: { ietf: 'en', tk: 'pps7abe.css' },
-          il_he: { ietf: 'he', tk: 'nwq1mna.css' },
-          mena_ar: { ietf: 'ar', tk: 'dis2dpj.css' },
+          il_he: { ietf: 'he', tk: 'nwq1mna.css', dir: 'rtl' },
+          mena_ar: { ietf: 'ar', tk: 'dis2dpj.css', dir: 'rtl' },
```

Comment:
make sure your tests cover both `ltr` and `rtl` use-cases

Reply:
Let's add tests for `rtl` as well

Reply:
still need rtl test

Reply:
@chrischrischris The rtl and ltr test are already there in the Unit tests.. we have added the dir property for the same in that config .

Reply:
You are correct! But I think one existing test would fail, maybe you can check - https://github.com/adobecom/milo/blob/main/test/utils/utils.test.js#L258

```js
setConfigWithPath('/ua/solutions');
expect(document.documentElement.getAttribute('dir')).null;
``` 

Based on what you set - `document.documentElement.setAttribute('dir', contentDir || config.locale.dir || new Intl.Locale(config.locale.ietf)?.textInfo?.direction || 'ltr');`, this will return `ltr` and we're expecting to be null

Reply:
@narcis-radu The test was completing successfully because it was only checking that if the dir attribute set fails it should not throw error and the value should be null.
I have added a check to verify that the Intl should run only if config.locale.ietf is set and if not it should not try to find textinfo from it and fallback to 'ltr'
Could you please review?